en/components/grids_templates/excel_style_filtering.md

```diff
@@ -450,6 +450,94 @@ In order to provide a custom loading template for the excel style filtering, we
 
 <div class="divider--half"></div>
 
+
+### External Excel Style filtering
+
+As you see at the demos above the default appearance of the Excel Style filtering dialog is inside the grid. So this dialog is only visible when configuring the fitlers. There is a way to make that dialog stay always visible - it can  be used outside of the grid as a standalone component. All you need to do is to add it in your html and set its `column` property.
```

Comment:
- 'inside the grid' should be 'inside the @@igComponent'.
- Remove double space 'can  be'.
- The last sentence "All you need..." seems to be repeated in other places. You could replace it here with something like 'In the demo below, the Excel style filtering is declared separately of the @@igComponent'.